core/retail-basket.tab.info

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "retail-basket",
+    "description": "This is a transnational data set which contains all the transactions occurring between 2010-12-1 and 2011-12-09 for a UK-based and registered non-store online retail. The company mainly sells unique all-occasion gifts. Many customers of the company are wholesalers.",
+    "title": "Online Retail",
+    "tags": ["marketing"],
+    "target": null,
+    "version": "1.0",
+    "collection": "UCI",
+    "instances": 914,
+    "missing": false,
+    "variables": 50,
+    "year": "2012",
+    "references": [
+        "Daqing Chen, Sai Liang Sain, and Kun Guo (2012) <a href='https://link.springer.com/article/10.1057/dbm.2012.17'>Data mining for the online retail industry: A case study of RFM model-based customer segmentation using data mining.</a> Journal of Database Marketing and Customer Strategy Management 19(3), pp. 197."],
+    "source": "<a href:'http://archive.ics.uci.edu/ml/datasets/online+retail'>http://archive.ics.uci.edu/ml/datasets/online+retail</a>",
```

Comment:
I missed this:
<a href:    ->   <a href=
Also, to bo consistent with other UCI data sets, make the source say "UCI ML Repository"
`"source": "<a href='http://...'>UCI ML Repository</a>",`